examples/model_compress/pruning/basic_pruners_torch.py

```diff
@@ -278,6 +278,8 @@ def trainer(model, optimizer, criterion, epoch):
         m_speedup.speedup_model()
 
     print('start finetuning...')
+    optimizer = torch.optim.SGD(model.parameters(), lr=0.01, momentum=0.9, weight_decay=5e-4)
+    scheduler = MultiStepLR(optimizer, milestones=[int(args.pretrain_epochs * 0.5), int(args.pretrain_epochs * 0.75)], gamma=0.1)
```

Comment:
I found the same code at line 147 and 148...

Reply:
yes, I copy from there...
This because `optimizer` might be modified in pruner, ref line 230, 258, then we need a new `optimizer` in finetuning stage.

Reply:
add a comment for these lines